currency-converter/Ballerina.toml

```diff
@@ -0,0 +1,5 @@
+[package]
+org = "wso2"
```

Comment:
In Samples also are we gonna change the org to "myOrg" as discussed in the meeting?

Reply:
no, we should ideally create a new org in ballerina central (to keep these samples as templates under that org), so we should ideally have that new org name in here. Since that is not finalized yet, kept it this way ftm